### PR TITLE
Add game state boolean ref docs

### PR DESCRIPTION
**Pull request by ganicke**
- [x] Add doc pages for **isRunning**, **isPaused**, **isGameOver**
- [x] Link up pages in _game_ card page
- [x] Fix api help paths and descriptions
- [x] Remove the `?` in the block text. Do we use this for conditions normally???
- [x] Add ``!_paused`` as a condition for  **isRunning**.

Would ``isRunning()`` not mean:

```typescript
export function isRunning(): boolean {
        return !_isGameOver && !_paused;
}
```

Block look:

image

RE #1677

### CODE REVIEW

File: docs/reference/game/is-running.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* a [boolean](/types/boolean) value that is `true` if the game is running or `false` if not.
+
+## Example
+
+If the game is currently running, end the game if button **B** is pressed.
+
+```blocks
+input.onButtonPressed(Button.B, function () {
+	if (game.isRunning()) {
+        game.gameOver()
```
Comment (pelikhan) [inline docs/reference/game/is-running.md]: You should be able to call gameOver as many times as you wish

File: libs/core/game.ts [hunk 2]
```diff
@@ -250,13 +250,12 @@ namespace game {
...
+     * Indicates if the game is still running. Returns `false` if the game is over or paused.
      */
-    //% weight=5 help=game/isrunning
-    //% blockId=game_isrunning block="is running?" blockGap=8
+    //% weight=5 help=game/is-running
+    //% blockId=game_isrunning block="is running" blockGap=8
     export function isRunning(): boolean {
-        let running: boolean;
-        return !_isGameOver;
+        return !_isGameOver && !_paused && (_img != undefined);
```
Comment (pelikhan) [inline libs/core/game.ts]: !!_img

**Comment by xuefengedu** [comment]
For `Do we use this for conditions normally???`, I just followed block of `isTouching()` and `isTouchingEdge()`, you can have a check, thanks.

**Comment by ganicke** [comment]
@xuefengedu - Ah, can't blame you for that. We might have some inconsistency in that group. It's best if the block ends with the punctuation though. We have this in **Input**:

image

**Pull request merged**